src/sass/components/_datasets.scss

```diff
@@ -0,0 +1,73 @@
+// Dataset Capture Panels
+// ======================
+
+.metecho-modal-inner-content {
+  display: grid;
+  overflow-y: hidden;
+
+  @media (min-width: $mq-large) {
+    grid-template-rows: var(--modal-content-rows, minmax(0, auto) 1fr);
+    max-height: 100%;
+  }
+}
+
+.metecho-scroll-panel-container {
+  overflow-y: hidden;
+
+  .slds-card {
+    display: flex;
+    flex-direction: column;
+    overflow-y: hidden;
+  }
+
+  .slds-card__body {
+    overflow-y: hidden;
+
+    > div {
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      overflow-y: var(--panel-overflow);
```

Comment:
Should this have a fallback value?

Reply:
@stacyk Is this something you'd be willing to address?

Reply:
@mirisuzanne Because this slds-card_body is used in a lot of places I wanted to leave it without a fallback.

Reply:
@stacyk The default fallback is `unset`, so I don't think this works to protect other locations? If you really only want to override it in one place, it might be best to do that where you are currently setting the variable? Because this variable has potential to unset any other values.

Reply:
@stacyk I think this is the final thing to address before we can merge this PR, if you have a minute today.